src/AppForeach.Framework.EntityFrameworkCore/DataType/EntityTypeBuilderExtensions.cs

```diff
@@ -1,9 +1,32 @@
-﻿using Microsoft.EntityFrameworkCore.Metadata.Builders;
+﻿using AppForeach.Framework.DataType;
+using AppForeach.Framework.DataType.Facets;
+using Microsoft.EntityFrameworkCore.Metadata.Builders;
 
 namespace AppForeach.Framework.EntityFrameworkCore.DataType
 {
     public static class EntityTypeBuilderExtensions
     {
-        public static void FromEntitySpecification<T>(this EntityTypeBuilder<T> ext) where T : class { }
+        public static void FromEntitySpecification<T>(this EntityTypeBuilder<T> builder, BaseEntitySpecification<T> entitySpecification) where T : class
+        {
+            //todo: make field name part of specification
+            foreach (var fieldSpecification in entitySpecification.FieldSpecifications)
```

Comment:
Mapping between entity specification and EF entity type builder is happening here.

Here is how client call looks like:

![image](https://github.com/user-attachments/assets/bb362203-3bfe-435d-be7a-29c5993ce47a)